Source/TestDafny/MultiBackendTest.cs

```diff
@@ -180,10 +180,13 @@ int ExpectedExitCode
     string expectFile = options.TestFile + ".expect";
     var commonExpectedOutput = await File.ReadAllTextAsync(expectFile);
 
+    var onlyCompilers = 
+      (Environment.GetEnvironmentVariable("DAFNY_INTEGRATION_TESTS_ONLY_COMPILERS") ?? "").Split(",");
+
     var success = true;
     foreach (var plugin in plugins) {
       foreach (var compiler in plugin.GetCompilers(DafnyOptions.Default)) {
-        if (!compiler.IsStable) {
+        if (!compiler.IsStable || onlyCompilers.Any() && !onlyCompilers.Contains(compiler.TargetId)) {
           // Some tests still fail when using the lib back-end, for example due to disallowed assumptions being present in the test,
           // Such as empty constructors with ensures clauses, generated from iterators
```

Comment:
FYI this comment really needs to move to the `IsStable => false` line of `LibraryBackend` instead - your change makes it even more inaccurate here :)